### DISCUSSION TITLE
Wild Crops in other dimensions (Twilight Forest)

### DISCUSSION THREAD

**Discussion by RudiRammlervonSommerlatte** [vectorwing/FarmersDelight#855] [Q&A]
Hello guys,
how would I go about making wild crops spawnable in other dimensions such as Twilight Forest? In the config files there are no options to add other dimensions and i'm not experienced enough to edit the mod file itself. Do you guys have any idea if this is even possible?
Thanks in advance!

**Comment by Aceplante** [comment]
As a twilight forest user, they already seem to do this.
